src/wp-admin/js/customize-controls.js

```diff
@@ -3975,7 +4007,6 @@
 				statuses     = this.container.find('.dropdown-status'),
 				params       = this.params,
 				toggleFreeze = false,
-				// eslint-disable-next-line func-style
```

Comment:
`// eslint-disable-next-line func-style`

Leave this comment line in the code to silence `Expected a function declaration` errors / warnings in ESLint.

Reply:
Thanks for the explanation! Now done.